js/notification.js

```
@@ -5,22 +5,26 @@ const path = require('path');
 
 const title = 'Time to Leave';
 
-function notify(msg)
+function notify(msg, actions)
 {
     const appPath = process.env.NODE_ENV === 'production'
         ? `${process.resourcesPath}/app`
         : path.join(__dirname, '..');
 
-    return new Promise(resolve =>
+    return new Promise((resolve, reject) =>
     {
         notifier.notify({
             title: title,
             message: msg,
             icon: path.join(appPath, 'assets/timer.png'), // Absolute path (doesn't work on balloons)
             sound: true, // Only Notification Center or Windows Toasters
-            wait: true
+            wait: true,
+            actions
+        },(err, data) =>
+        {
+            if (err) reject(err);
```

do we just ignore the error, if it happens?